### ISSUE TITLE
Armor texture glitch 1.12.2

### ISSUE THREAD

**Issue by Brudaouas** [TheIllusiveC4/ConstructsArmory#261]
![2021-02-11_03 30 50](https://user-images.githubusercontent.com/78900484/107615203-be712380-6c19-11eb-9b62-2ea9126b86ab.png)
![2021-02-11_03 31 37](https://user-images.githubusercontent.com/78900484/107615216-c335d780-6c19-11eb-99c0-604a4e0de8d2.png)

here is what it looks like, I know this has been fixed before, but I don't know what would be causing this for me, if you need a photo of my mod list ill put it in. As far as that I don't know what would be causing this, I am using the latest version of the mod for 1.12.2 and the recommended forge for 1.12.2. please let me know if I need to provide anything

**Comment by TheIllusiveC4** [comment]
Yes, please provide a modlist.

**Comment by Brudaouas** [comment]
![Mc Mod List](https://user-images.githubusercontent.com/78900484/107699399-088eef00-6c84-11eb-832f-36447278b5f7.PNG)

here is the mod list, if you need a better photo of it or a more in-depth detail I can send the folder itself. I have used this mod before with a mod list like this, and it worked flawlessly, just with this mod list the textures mess up. *Edit* in the original photo that started this I was on the latest version of the mod and it still had this issue, but I had to roll it back to an older version for that version I use on a small dedicated server with my friend, However, this issue lasted to the lasted version. as well as conflicted heavily with crafting tweaks 2. so I had to remove crafting tweaks 2 temporarily to just test to see if I could get the textures fixed.

**Comment by TheIllusiveC4** [comment]
Try removing Modern Warfare `mw-2.0-0.4.4.6_mc1.12.2`.

**Comment by Brudaouas** [comment]
Wow didn't think that mod like that would cause an issue like that. But it did and I tested it and it is fixed now. Thank you for the help @TheIllusiveC4. *Edit* Immediately after this I looked for a compatibility mod, for it, there isn't one. And I stumbled upon a post about this on your form you closed last year. I guess this has always been an issue, ill have to report it to the creator of vics MW if he actually had issues tab.
